docgen/src/guide/Sorting_and_filtering.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Sorting and filtering items
+mainTitle: Guide
+layout: main.pug
+category: guide
+navWeight: 78
+---
+
+A frequent question that comes up is "How do I sort or filter the items of my widget".
+
+For this, widgets and connectors that are handling items expose a `transformItems` prop. This prop is a function that has the `items` provided 
+prop as a parameter. It will expect in return the `items` prop back. 
+
+The following example will show you how to change the default sort order of the [`<RefinementList/>`](widgets/RefinementList.html) widget. Keep in mind
+that you can apply the same strategy on every widget that handle a list of `items`. 
```

Comment:
I can enumerate them if you think it's better with it. We just need to not forgot to update this guide if the list grows.